students/aravichander/lesson08/test_circle_class.py

```diff
@@ -0,0 +1,6 @@
+def test_circle_size():
```

Comment:
good start, but more tests! 

what about setting the diameter? and creating one from a diameter? and adding them together, and....

Reply:
Remember test driven development -- you should have a test for the feature you are working on as soon as you start to write the code....